code/modules/materials/Mat_Processing.dm

```diff
@@ -71,10 +129,6 @@
 		if (!istype(W))
 			return
 
-		if(isExploitableObject(W))
```

Comment:
is this proc still needed then?

Reply:
It seems to still be used by the nano-crucible in the same file.